androidHyperskillApp/src/main/java/org/hyperskill/app/android/core/injection/AndroidAppComponentImpl.kt

```diff
@@ -287,4 +289,7 @@ class AndroidAppComponentImpl(
 
     override fun buildProgressesDataComponent(): ProgressesDataComponent =
         ProgressesDataComponentImpl(this)
+
+    override fun buildPlatformTopicsRepetitionsComponent(recommendedRepetitionsCount: Int): PlatformTopicsRepetitionComponent =
+        PlatformTopicsRepetitionComponentImpl(this, recommendedRepetitionsCount)
```

Comment:
`recommendedRepetitionsCount` -> _The corresponding parameter in the supertype 'AndroidAppComponent' is named 'buildPlatformTopicsRepetitionsComponent'. This may cause problems when calling this function with named arguments._

Обычно мы `buildPlatform***Component` ложим рядом с shared имплементацией и разделяем комментарием, чтобы было более читабельнее.

```kotlin
/**
 * Topics repetitions component
 */
override fun buildTopicsRepetitionsComponent(): TopicsRepetitionsComponent =
    TopicsRepetitionsComponentImpl(this)

override fun buildPlatformTopicsRepetitionsComponent(recommendedRepetitionsCount: Int): PlatformTopicsRepetitionComponent =
    PlatformTopicsRepetitionComponentImpl(this, recommendedRepetitionsCount)
```